### PR TITLE
Icons: add star and radio button icons, organize folder

### PR DESCRIPTION
**Pull request by ryelle**
This PR updates the icon components, pulling each into their own folder. I've also added the radio selected & unselected icons, and the star icon (used for Featured Product). Additionally, we've switched to using the Material checkbox icons directly, rather than a masked SVG (just to save some rendering). This matches the radio icons, but might appear slightly larger now – if so, will fix in CSS in a following PR.

### Screenshots

New checkmarks:

screen shot 2019-01-09 at 12 15 38 pm

Icons for blocks:

screen shot 2019-01-09 at 12 29 10 pm

### How to test the changes in this Pull Request:

1. Look at all block icons, there should be no change
2. Add a Products by Category or Hand-picked Products block, the selection UIs should have working checkmark icons.

### CODE REVIEW

File: assets/js/components/icons/star.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ * External dependencies
+ */
+import { Icon } from '@wordpress/components';
+
+export default () => (
+	<Icon
+		icon={
+			<svg
+				xmlns="http://www.w3.org/2000/svg"
+				width="18"
```
Comment (timmyc) [inline assets/js/components/icons/star.js]: I noticed all the other icons were set to 24px - any reason for this one being 18?

**Review by timmyc — APPROVED** [review]
Code LGTM 👍

**Review by LevinMedia — APPROVED** [review]
LGTM! - 24px is fine for the checkbox size

File: assets/js/components/icons/star.js (hunk 1 above)
Comment (ryelle) [reply assets/js/components/icons/star.js]: No idea, but that's what I get from [the material icons website.](https://material.io/tools/icons/) I think it's fine, it'll scale up to fit the dimensions needed.

**Pull request merged**